README.md

```diff
@@ -59,15 +59,26 @@ See below for documentation.
 
 ## Installation
 
-Add `tesla` as dependency in `mix.exs`
+Add `tesla` as dependency in `mix.exs`:
 
 ```elixir
 defp deps do
-  [{:tesla, "~> 1.2.0"},
-   {:jason, ">= 1.0.0"}] # optional, required by JSON middleware
+  [
+    {:tesla, "~> 1.2.0"},
+    {:hackney, "~> 1.4.0"}, # optional, but recommended adapter
```

Comment:
This requirement won't match the latest hackney versions.

Reply:
Hmm latest is `1.14.3` and .... ok I'm blind. thanks!